File: 3/4/Person.java
public class Person {
    public String first_name;
    public String last_name;
    private int age;
    
    public Person(String fname, String lname, int age){
        this.first_name = fname;
        this.last_name = lname;
        this.age = age;
    }

    public int getAge() {
        return this.age;
    }

    public void ageUp(){
        this.age++;
    }
    public String getFullname(){
        return this.first_name +" " + this.last_name;
    }

    public void printFullName(){
        System.out.println(this.getFullname());
    }
} 

File: 3/1/person.py
class Person:
    def __init__(self, fname: str, lname: str, age: int) -> None:
        self.first_name: str = fname
        self.last_name: str = lname
        self.__age = age
        return None

    def getAge(self) -> int:
        return self.__age

    def ageUp(self) -> None:
        self.__age += 1

    def getFullname(self) -> str:
        return f"{self.first_name} {self.last_name}"

    def printFullName(self) -> None:
        print(self.getFullname())
